### PR TITLE
Used endpoints to fetch the prices set in stripe and display them on …

### PR DESCRIPTION
**Pull request by Kartik-M24**
…the home using the wrapper

### CODE REVIEW

**Comment by Kartik-M24** [comment]
image
I tested to check if the additional info and original prices could also be displayed. 
image

File: client/src/services/AppData/AppDataService.ts [hunk 1]
```diff
@@ -1,41 +1,86 @@
...
+              break
+            case "returning_member":
+              displayName = "Returning Member"
+              break
+            case "new_non_student":
+              displayName = "New Non-UoA Student"
+              break
+          }
+          if (data.originalPrice != null) {
+            data.originalPrice = "$" + data.originalPrice
```
Comment (bcho892) [inline client/src/services/AppData/AppDataService.ts:66]:
```suggestion
            data.originalPrice = `$${data.originalPrice}`
```

File: client/src/pages/Home/sections/PricingSection.tsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+import { Prices } from "services/AppData/AppDataService"
 import HomeSectionHeading from "./utils/HomeSectionHeading"
 import HomeSectionWrapper from "./utils/HomeSectionWrapper"
 import PricingCard from "components/generic/PricingCard/PricingCard"
 import PricingBanner from "components/generic/PricingBanner/PricingBanner"
 
 interface IPricingSection {
   note?: string
-  pricings: Pricing[]
+  // TODO: Update this type to the type of data returned by the `fetch-users` endpoint
```
Comment (bcho892) [inline client/src/pages/Home/sections/PricingSection.tsx:10]: remove todo

**Review by bcho892 — COMMENTED** [review]
LGTM, just some small changes, i suggest you read the workflow errors and fix them

File: client/src/services/AppData/AppDataService.ts [hunk 3]
```diff
@@ -1,41 +1,86 @@
...
+            case "non_uoa_student":
+              displayName = "Non-UoA Student"
+              break
+            case "returning_member":
+              displayName = "Returning Member"
+              break
+            case "new_non_student":
+              displayName = "New Non-UoA Student"
+              break
+          }
```
Comment (bcho892) [inline client/src/services/AppData/AppDataService.ts:64]: You might want to extract this into a util function and use it in `client/src/components/composite/SignUpForm/Sections/PaymentSection.tsx`

**Review by bcho892 — APPROVED** [review]

**Pull request merged**